common/include/joy.h

```diff
@@ -36,6 +36,7 @@ extern void joy_init();
 extern void joy_close();
 const d_event_joystick_axis_value &event_joystick_get_axis(const d_event &event);
 extern void joy_flush();
+extern bool joy_translate_menu_key(const d_event &event);
```

Comment:
`extern` is unnecessary here.  The existing declarations have not been stripped, but there is no need to include it in new ones.

Reply:
OK. Again, I tried to follow the surrounding code :)

Reply:
That's fine.  I appreciate the attempt to maintain consistency.  Rebirth hasn't been truly consistent for a long time though. ;)